### PR TITLE
Tools

### PR DESCRIPTION
**Pull request by andrewhayward**
Adding in the 'tools' section of the site.

### CODE REVIEW

File: make_mozilla/tools/models.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    ('2', 'Live'),
+    ('1', 'In Progress'),
+    ('0', 'Coming Soon'),
+)
+
+
+class Tool(models.Model):
+    name = models.CharField(max_length = 255)
+    strapline = models.CharField(max_length = 100)
+    url = models.URLField(blank = True)
```
Comment (rossbruniges) [inline make_mozilla/tools/models.py:16]: whenever you use blank = True also include null = True.

Null pertains to the DB, blank the built in admin and form validations (if I remember correctly)

File: make_mozilla/tools/models.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    slug = models.SlugField()
+    description = models.TextField(blank = True)
```
Comment (rossbruniges) [inline make_mozilla/tools/models.py:18]: As above

File: make_mozilla/tools/models.py [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    featured = models.BooleanField(default = False)
+    new = models.BooleanField(default = True)
+    status = models.CharField(max_length = 1, choices = TOOL_STATUS_CHOICES)
+    logo = models.ImageField(upload_to = 'tools', storage = FileSystemStorage(**settings.UPLOADED_IMAGES))
```
Comment (rossbruniges) [inline make_mozilla/tools/models.py:22]: Some tools probably won't have a nice icon for launch - so we should set this field to be optional.

**Comment by rossbruniges** [comment]
General comment - please send the pull request to the development branch.

Cheers

**Comment by andrewhayward** [comment]
Will fix models, and resubmit to the dev branch.

**Comment by rossbruniges** [comment]
So I think in the future the pull requests update themselves once you re-commit to the branch that you're looking to push in, meaning (I think) no need to delete them - unless you need to do that for pushing into the different branch...

**Pull request closed without merging**